frontend/templates/vulkan/server.tsx

```diff
@@ -0,0 +1 @@
+export { getServerSideProps } from '@templates/page/server';
```

Comment:
Just to be sure, you're using the `getServerSideProps` from the `page` template just as a placeholder? Although `page` might seems like a generic concept, it's just a specific template to handle custom pages (such as the store home page, and any other one-off page made of sections). To be clear, it's not a template to be used for building up other templates

Reply:
That's exactly the kind of thing I wanted to know; thanks. I assumed it was relatively generic. Is there a good place to get the props required by the layout? I didn't want to have to do a bunch of copy-paste to generate the correct data for it.

Reply:
`templates/vulkan/server.tsx` is the correct place where to define the `getServerSideProps` for the `vulkan` template. But you need to define the function yourself, not import it from another template, as each template would have its own data requirement. Looks like what you did with the troubleshooting template is the correct approach 👍 Some props will be the same for every template (e.g. the layouts data), but I'd expect every template to have also its own specific props based on the data dependencies of the template